src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java

```diff
@@ -135,6 +137,36 @@ public boolean getDefaultFormProcessing() {
 
         };
     }
+    
+    /**
+     * Compute the native bounds from the native CRS. Acts as an alternative to computing the bounds
+     * from the data itself.
+     */
+    AjaxSubmitLink computeBoundsFromSRS(final Form refForm, 
+            final EnvelopePanel nativeBoundsPanel) {
```

Comment:
You may want to look at your code formatting settings, think we have a 100 col width here?